pipelines/WeCount-ODC.json5

```diff
@@ -0,0 +1,99 @@
+{
+    type: "fluid.pipelines.WeCount-ODC",
+    parents: "fluid.pipelines.withOctokit",
+    elements: {
+        ODCConfig: {
+            type: "fluid.component",
+            scrapeURL: "https://data.ontario.ca/dataset/covid-19-assessment-centre-locations"
+        },
+        gitConfig: {
+            type: "fluid.component",
+            repoOwner: "inclusive-design",
+            repoName: "covid-assessment-centres",
+            branchName: "main",
+            ODCFolder: "ODC/",
+            mergedFolder: "merged/"
+        },
+        ODCCoordinates: { // generates members dataFileName, date, downloadURL
+            type: "fluid.dataMonitor.getODCFileCoordinates",
+            scrapeURL: "{ODCConfig}.scrapeURL",
+            folder: "{gitConfig}.ODCFolder"
+        },
+        hasNewODCFile: {
+            type: "fluid.dataPipe.gitFileNotExists",
+            repoOwner: "{gitConfig}.repoOwner",
+            repoName: "{gitConfig}.repoName",
+            branchName: "{gitConfig}.branchName",
+            filePath: "{ODCCoordinates}.data.dataFileName"
+        },
+        ODC: {
+            type: "fluid.fetchUrlCSV",
+            hasFile: "{hasNewODCFile}.data", // Fake dependency to short-circuit pipeline if the file is not new
+            url: "{ODCCoordinates}.data.downloadURL"
+        },
+        WeCount: {
```

Comment:
Is the next step to generate synthetic accessibility data in this "wecount" step rather than fetching a static CSV file?

Reply:
Thanks for the reminder - I need to apply the "synthetic" pipeline grade from forgiving-data